### PR TITLE
Add a question regarding rel attribute in accessibility quiz.ts

### PR DESCRIPTION
**Pull request by sanjaypinna**
## Summary of changes

This PR replace a duplicate question (which already exist in line number 346 -356) with a new one regarding rel attribute in accessibility quiz.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

Relates to #374

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -378,14 +378,14 @@ const accessibilityQuiz = [
   },
   {
     Question:
-      "Which of the following HTML tags does NOT use the autocomplete attribute?",
-    Answer: "<html>",
+      "Which of the following HTML tags does NOT use the rel attribute?",
+    Answer: "<link>",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:382]: the rel attribute is valid for link elements. So you will to come up with a different answer here

```suggestion
    Answer: "<input>",
```

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -378,14 +378,14 @@ const accessibilityQuiz = [
...
     Distractor1: "<input>",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:383]: it looks like input does not accept the rel attribute according to MDN

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

```suggestion
    Distractor1: "<link>",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/accessibility-quiz.ts (hunk 2 above)
Comment (sanjaypinna) [reply src/data/accessibility-quiz.ts:383]: @jdwilkin4 thank you fixed it

**Comment by sanjaypinna** [comment]
@jdwilkin4 thanks you for pointing out the mistake I made the changes as requested, you can merge it

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**